core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -246,24 +329,36 @@ private boolean isIntermediateTopic(final String topic) {
         return options.valuesOf(intermediateTopicsOption).contains(topic);
     }
 
-    private void deleteInternalTopics(final ZkUtils zkUtils) {
-        System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+    private void maybedeleteInternalTopics(final ZkUtils zkUtils) {
+        Boolean dryRun = options.has(dryRunOption);
+
+        if (!dryRun) {
+            System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+        } else {
+            System.out.println("Following internal/auto-created topics will be Deleted for application " + options.valueOf(applicationIdOption));
```

Comment:
Added the check because, in dry-run we only print actions, and not do the action.
As this statement looks like Deleting all internal/auto-created topics for application g1 is like it is an action.